converters/google/convert.go

```diff
@@ -122,6 +124,7 @@ type Converter struct {
 
 	cfg *converter.Config
 
+	// Talk to GCP resource manager. This field would be nil in offline mode.
```

Comment:
I think we should make Converter into an interface and add offline and online implementations.